frontend/pages/beta/create-new-model.tsx

```diff
@@ -0,0 +1,97 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Card from '@mui/material/Card'
+import Divider from '@mui/material/Divider'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Radio from '@mui/material/Radio'
+import RadioGroup from '@mui/material/RadioGroup'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import { useState } from 'react'
+
+import Wrapper from '../../src/Wrapper'
+
+type NewModelData = {
+  teamName: string
+  modelName: string
+  description: string
+  accessibility: 'public' | 'private'
+}
+
+export default function CreateNewModelWizard() {
+  const [teamName, setTeamName] = useState('')
+  const [modelName, setModelName] = useState('')
+  const [description, setDescription] = useState('')
+  const [accessibility, setAccessibility] = useState<NewModelData['accessibility']>('public')
+
+  function onSubmit(event) {
+    event.preventDefault()
+    const formData: NewModelData = {
+      teamName,
+      modelName,
+      description,
+      accessibility,
+    }
+    // TODO - after new model page is implemented, forward this data
+    console.log(formData)
+  }
+  return (
+    <Wrapper title='Create a new Model' page='marketplace'>
```

Comment:
Do you mean the overall form? It's based on the wireframe designs, I think it just looks odd because the wireframes are much more compact width wise whereas you're viewing it on a wider resolution. I've discussed this with people before but we actually agreed having it wider looks better. 

My reservation about reducing the width is that it might look odd if some pages have a smaller width than others; this way we can keep it consistent. Thoughts?